packages/kotti-ui/source/kotti-style/forms/_misc.scss

```diff
@@ -73,10 +73,7 @@
 }
 
 // Disabled fields
-.form-input,
-.form-checkbox,
-.form-switch,
-.form-radio {
```

Comment:
This class is still used in `src/user-panel-frontend/src/components/basket/wizard/BasketWizardPostProcessingSingleSelect.vue`